inliner.go

```diff
@@ -0,0 +1,26 @@
+package cidutil
+
+import (
+	cid "github.com/ipfs/go-cid"
+	mhash "github.com/multiformats/go-multihash"
+)
+
+// Inliner is a cid.Builder that will use the id multihash when the
+// size of the content is no more than limit
+type Inliner struct {
+	cid.Builder
+	Limit int
```

Comment:
Should probably be documented (inclusive? exclusive?).

Reply:
Do you mean "Limit"? I said above "size of the content is no more than limit" which should imply inclusive.

Reply:
Ah. I didn't see that. Yes, that should be sufficient.